config/initializers/simple_form.rb

```diff
@@ -0,0 +1,165 @@
+# Use this setup block to configure all options available in SimpleForm.
+SimpleForm.setup do |config|
```

Comment:
how many of inititalizers you plan to add? you have 2 here. leave only one for bootstrap